script/publish_persistent_udfs

```diff
@@ -33,14 +33,10 @@ parser.add_argument(
     "--project-id", default="moz-fx-data-derived-datasets", help="The project ID."
 )
 parser.add_argument(
-    "--dataset",
-    default="udf",
-    help="The name of the dataset the persistent UDFs will be stored in.",
-)
-parser.add_argument(
-    "--udf-dir",
-    default="udf/",
-    help="The directory where declarations of temporary UDFs are stored.",
+    "--udf-dirs",
+    nargs="+",
+    default=["udf/", "udf_js/"],
```

Comment:
It might be handy to specify multiple directories with UDFs at once.

Reply:
Do we lose the ability to specify the target dataset? By making this change, we assume that the dataset is named the same as the directory?

I might be okay with that, but we should update the help string to make this more clear. I do like the idea that running the script without arguments handles everything we want to do during schema deploys.

Reply:
Yeah, the target datasets would be the same as the names of those directories. I updated the help string to make this clearer.